packages/onft-evm/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@layerzerolabs/onft-evm",
+  "version": "0.0.1",
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LayerZero-Labs/devtools.git"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.cjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "artifacts/contracts/**/!(*.dbg).json",
+    "contracts/**/*"
+  ],
+  "scripts": {
+    "build": "forge build",
+    "build-contracts": "forge build && $npm_execpath hardhat compile --no-typechain --no-copy-artifacts && $npm_execpath hardhat size-contracts --no-compile",
```

Comment:
This naming convention seems a bit random - i'd stick to using : as delimiter and keep the hierarchy (pseudocode):

`test: "test:forge && test hardhat"`

Then the convention for what the scripts do generally is:

compile for contracts
build for typescript

Build task depends on compile so when you run pnpm build it will run just fine